content/en/security/cloud_security_management/setup/without_infrastructure_monitoring.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Setting Up CSM without Infrastructure Monitoring
+---
+
+You can set up CSM without infrastructure monitoring, so you can save on costs.
```

Comment:
Let's not phrase it this way. You ca say something in the lines of:

You can set up CSM with or without Infrastructure Monitoring.

Reply:
@rajat-luthra-ddog Can we talk a bit about why they would want to? "With or without" is a bit misleading, because the topic focuses on doing it without.